app/main.py

```diff
@@ -1 +1,33 @@
-# write your code here
+class Animal:
+
+    alive = []
+
+    def __init__(
+            self,
+            name: str,
+            health: int = 100,
+            hidden: bool = False
+    ) -> None:
+        self.health = health
+        self.name = name
+        self.hidden = hidden
+        self.alive.append(self)
+
+    def __repr__(self) -> str:
+        return str(
+            {"Name": self.name, "Health": self.health, "Hidden": self.hidden}
+        ).replace("'", "")
```

Comment:
Why do you need replace? Remove it

Reply:
without replace tests will fail:
`E       assert "[{'Name': 'K...n': False}]\n" == '[{Name: King...en: False}]\n'
E         - [{Name: King Lion, Health: 100, Hidden: False}, {Name: Bagira, Health: 100, Hidden: False}, {Name: Susan, Health: 100, Hidden: False}]
E         + [{'Name': 'King Lion', 'Health': 100, 'Hidden': False}, {'Name': 'Bagira', 'Health': 100, 'Hidden': False}, {'Name': 'Susan', 'Health': 100, 'Hidden': False}]
E         ?   +    +  +         +  +      +       +      +           +    +  +      +  +      +       +      +           +    +  +     +  +      +       +      +
`

Reply:
Anyway, the solution could be done in a cleaner way. Use f-strings instead of str constructor and .replace()